Add staking phase 1.1 info (#165)
Zilliqa/dev-portal: docs/staking/phase1/staking-phase11-notice.md

---
id: staking-phase11-notice
title: Staking phase 1.1 migration notice
keywords: 
- staking
- general information
- ssn
- seed node
- zilliqa	
- migration
description: Staking phase 1.1 migration notice
---

:::important

**Staking phase 1 migration**

We will be performing a contract migration from staking phase 1 to phase 1.1. The migration will happen on Tuesday 11th May 2021. Phase 1 contract will be frozen permanently at around 04:45 UTC.
The migration may take up to 7 days to complete. We will aim to open staking as soon as the migration is completed.
:::

## Staking phase 1.1 

As proposed in [ZIP-19](https://github.com/Zilliqa/ZIP/blob/master/zips/zip-19.md), Staking phase 1.1 consists of the following changes
- Transfer of stake from one account to another account
- Proper deletion of empty maps to prevent contract states bloating
- Removal of user-defined ADT in `AssignStakeRewards` transition
- Staking parameters change

The migration to phase 1.1 will involve the following: 
1. Freezing of phase 1.0 contracts
2. Deployment of new proxy and ssnlist contracts
3. Migration of states and funds from phase 1 to phase 1.1

## Important note

> - The Contracts address will be change after the migration. Wallets and explorer will need to point to new contract addresses.

Reviewer: ```suggestion
- The contract addresses will be change after the migration. Wallets and explorer will need to point to new contract addresses.
```

- No staking activities can be performed during the migration 
- No action is needed for **delegators**

> - For **wallets, explore and node operator**, you will need to switch to the new contract addresses after the migration. All transition name and parameter in the contracts remain the same.

Reviewer: ```suggestion
- For **wallets, explorers and node operators**, you will need to switch to the new contract addresses after the migration. All transition names and parameters in the contracts remain the same.
```

> - Stake rewards will be retroactively be rewarded in full to all delegators and node operators for the period of migration

Reviewer: ```suggestion
- Stake rewards will be retroactively rewarded in full to all delegators and node operators for the period of migration
```

- We will also update Zillion staking portal to point to Phase 1.1 contracts

> ## Contracts information

Reviewer: ```suggestion
## Information on contracts
```

> For wallets and explorers, we will aim to let you know the phase 1.1 mainnet staking contract addresses as soon as we can. In the meanwhile, we have migrated the testnet staking contract

Reviewer: ```suggestion
For wallets and explorers, we will aim to let you know the phase 1.1 mainnet staking contract addresses as soon as we can. In the meantime, we have migrated the testnet staking contract
```

from phase 1 to phase 1.1. Please refer to [this page](staking-general-information) for Phase 1.1 testnet contracts.

## Changes to staking parameters (Mainnet)

> With mainnet `v8.0.0` upgrade, block time will be reduced. As such we will be making the following staking parameters changes

Reviewer: ```suggestion
With mainnet `v8.0.0` upgrade, block time will be reduced. As such, we will be making the following staking parameter changes
```

| Parameter | Phase 1.0 | Phase 1.1 |
|-------------- | ------------- | --------- |
| 1 Cycle duration | ~27 hours | ~24 hours | 
| Block per cycle | 1800 | 2500 | 
| Reward per cycle | 1,980,000 $ZIL | 1,760,000 $ZIL |
| Unbonding period | 24,000 final blocks | 35,000 final blocks (~2 weeks) |

For wallets and explorers, you may need to adjust your UI to reflect the changes in the parameters. 

## Migration duration

| | Duration | 
|-------------- | ------------- | 
| Starting Date | Tuesday 11th May 2021 04:45 t * |
| Ending Date | Tuesday 18th May 2021 05:00* |

> *In the event of migration ending early than scheduled, we will resume the staking activities at the early possible time

Reviewer: ```suggestion
*In the event of migration ending earlier than scheduled, we will resume the staking activities at the earliest possible time
```
